README.md

```diff
@@ -1,14 +1,164 @@
 # Community
 
-## How to contribute?
+## What is rust-vmm?
 
-### Adding a new crate to rust-vmm
+rust-vmm is an open-source project empowering the community to build custom
+Virtual Machine Monitors (VMMs) and Hypervisors. It provides a set of
```

Comment:
Well @raduweiss you know I like to change my mind often. Maybe it can be used for hypervisors as well. We don't know and I don't want to be so restrictive here.

Reply:
Ok ok.